### ISSUE TITLE
RIR Data": Error Not the correct number apnic. Should be 51371, but was 51370

### ISSUE THREAD

**Issue by Babola123** [HomeITAdmin/nextcloud_geoblocker#114]
I currently have a problem updating the database with the service RIRData (Local). I have tried it again and again the last few days, but always get the same error.

`The database is corrupted. Please run the update again. Last error message: Not the correct number of entries for IPv4 in the "apnic" region. Should be 51371, but was 51370.`

is there a problem at the moment?

My machine

Intel NUC i3 
Proxmox 7.3-4 
LXC Container Ubuntu 22.04.1 
Nextcloud 25.0.3

**Comment by HomeITAdmin** [comment]
Hi,
I cannot reprodruce the issue, here it is running fine.
Can you try resetting the database over the command line and then initializing the db again:
`sudo -u %NEXTCLOUD_INSTANCE_USER% php occ geoblocker:localization-service:reset-db 2`

**Comment by Babola123** [comment]
Hey and thanks for your reply. 

I just triggered the syncronization again without resetting the database and now it works again. It seems that there was a temporary problem.

I have compared the data under /var/www/nextcloud/apps/geoblocker/3rdparty/rir_data/

Before today's sync there were only 4 files
afrinic.txt
apnic.txt
arin.txt
ripencc.txt


and now there are 5 files

afrinic.txt
apnic.txt
arin.txt
ripencc.txt
lacnic.txt


The problem is solved for me.

Thanks for your app it is a great work.
